dev/type_coercion_udf.py

```diff
@@ -0,0 +1,134 @@
+import sys
```

Comment:
Let's add a license header. Also, this requires `pyspark` is in your Python path and Spark has to be built - would have to be explained.

If you put this under `python` directory, since python interpreter adds its current working directory to its path, it won't require that step. This is one alternative too.